src/scss/components/_button.scss

```diff
@@ -56,22 +56,8 @@
 		opacity: 1;
 	}
 
-	&::after {
-		content: "";
-		position: absolute;
-		border-radius: 5px;
-		width: 100%;
-		height: 100%;
-		top: 0;
-		left: 0;
-		background: helpers.$gradient-yellow-dark;
+	&:active::before {
 		transform: matrix(1, 0, 0, -1, 0, 0);
```

Comment:
what the hell is it?

can you understand what this string does if you see it in some other, (not yours) code?
I doubt it.
I told you, don't use code which you don't understand, or research code completely so that you coluld fully understand it.

all this shit I can write down like `transform: scale(-1);`

Reply:
Done